static-assets/components/cstudio-common/common-api.js

```diff
@@ -2157,16 +2157,14 @@ var nodeOpen = false,
                   .height()
                   .toString()
               );
-              $(dialog).height(60);
+              $(dialog).css('cssText', $(dialog).attr('style') + 'height: 65px !important');
```

Comment:
This is strange. Why not set the height directly? if not with `.height()`, with `.css('height', ...)`

Reply:
Height style needs to be important for it to work, and jquery doesn't support usage of important with `.height` or `.css('height', ...)`